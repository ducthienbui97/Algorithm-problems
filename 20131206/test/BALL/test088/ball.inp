9 354571 638570 682207 137003
304119 812357 509985 238190 602809 928102 679749 910373 821045 171416 823638 685766 742737 639640 515137 443691 55579 992729 560563 302181 873009 853566 429876 727925 395788 273699 122070 668638 227966 975405 620089 278288 497162 871482 682504 372509 229191 110848 509114 387023 367450 630438 65042 411026 818316 974761 363126 356709 536891 732727 351128 195446 572876 462549 589636 598037 110103 15678 996113 960414 791840 384932 817211 258894 787456 585243 466330 490311 305240 583381 913991 269831 792720 477880 570023 816507 674298 441442 469737 963174 522597 
455087 675250 118723 165355 535078 851435 515926 534941 221315 57703 918722 2515 89157 705660 223407 449316 93578 835135 625793 342319 96399 377385 231623 118267 833238 403366 122702 993071 438761 253609 722635 534519 799835 75766 6644 266687 412726 543456 356447 288277 69286 86219 578610 505710 37910 857242 778366 408380 455941 575427 115682 315463 890287 916090 456091 54496 566855 676398 537902 672066 137469 568000 417428 38783 521326 439101 118490 375659 138182 120908 736263 853938 301450 593993 251715 677738 353670 991577 791620 338375 24177 
677471 65561 321695 992815 903795 811169 169055 547084 171143 25402 309607 361158 407122 282794 655042 651168 503881 706518 676140 415582 608970 518876 442498 436278 322362 304365 801978 945565 151201 571655 144357 813013 822375 803859 52075 80415 166210 165317 525201 342333 549058 712876 223964 260813 772922 718813 130024 88314 13023 624146 385087 369401 111557 472215 533508 596122 961315 738427 837430 944007 149281 110958 901775 527578 575688 784835 499152 506978 200018 72448 500741 704707 491892 13769 283111 746218 653473 710258 108148 992938 195153 
537245 740168 601706 903960 220788 401309 677328 185960 841486 482533 2629 359264 407674 268119 671526 653339 299428 378004 476796 963035 572626 152733 456643 373283 533734 35189 766145 879649 907773 904577 585800 136791 582518 880139 670490 467798 911351 836335 228762 489328 654878 663712 390121 621605 77399 959191 118527 110687 405818 434652 490335 554977 519265 856816 705368 402661 361150 591083 266506 713319 918360 950091 826783 769546 347121 818236 641142 816951 258416 699755 60844 337795 871466 351218 281417 139022 172787 769843 84401 98507 996552 925188 325796 473981 680642 774404 380510 43284 620265 548041 745318 27813 836270 294864 98943 520445 8742 13729 815453 807955 451147 442620 941391 676207 309694 815282 284849 928181 708786 637058 601782 381206 724345 534501 267238 903341 808952 716234 198798 559831 374680 8073 532209 985757 109840 696703 660928 223891 734335 357565 140212 786793 671916 995954 178004 128329 408112 110692 184756 535167 503637 274680 390367 519671 236192 273304 507433 352627 389866 756152 221875 912604 578197 1926 877359 367536 984491 219586 910545 143924 544336 371904 77157 638764 545596 934028 340192 847369 437928 185050 482658 500373 992867 975202 723934 790577 195621 242643 420549 252824 313712 625900 637008 997067 147204 407789 125676 145427 352585 361471 637368 566858 119537 467756 297118 440686 974535 372716 98186 211628 372428 896031 975742 916178 363453 723121 123513 693626 772727 746137 153714 275402 621589 439655 79225 24686 659114 63799 989780 796132 268734 853752 555067 46833 937310 642223 374472 708227 153963 568819 645264 723543 747961 68431 531589 85532 236682 715923 985904 263997 875303 332771 34361 914617 771310 496799 690132 490318 360868 682045 573541 176673 255962 58669 93653 850914 731569 455955 958340 993428 773910 177394 939430 402081 495551 327979 429026 433055 700032 527944 582429 95903 379091 758944 968700 287117 957324 368796 314650 106507 491599 756443 538881 359837 783879 792533 671844 175937 851500 644789 95954 260133 982750 732646 380608 580958 837926 910473 359275 489198 785135 523540 804908 95935 937160 584494 594980 633209 814064 511608 523876 420566 265079 452300 364881 383446 905711 386689 245641 724072 810843 644514 255786 881360 279743 614785 15637 895807 352059 628747 878876 858777 772195 652049 300537 358622 452349 876220 651139 101533 103971 820551 139150 66195 297609 375973 900308 626158 295728 645063 206620 566479 715615 788944 704197 613412 868403 788471 496875 105168 422363 254496 576328 653353 728342 86574 849997 181927 394889 525879 755730 316990 667414 703520 197780 144398 541876 658618 295138 425522 843838 525963 161599 425624 814335 120543 418416 838010 842420 852759 751872 93276 595663 813998 65287 311157 844031 391986 945099 541594 353329 564392 948808 688521 765783 652057 787815 967254 866480 701447 233397 340523 665652 859432 863958 999670 659163 98443 279360 386005 578398 66537 385075 461207 200063 469834 89025 608673 592760 464574 846718 285083 624215 389729 194137 696757 415188 70434 460262 489345 831985 834099 224296 212192 274587 832286 346558 999075 763354 382059 245612 511147 175847 644224 539760 479716 524201 643035 695839 697980 377543 90184 301661 840088 3179 484924 538500 981610 206943 931160 795549 119375 762278 204852 263274 863855 825277 120352 175624 191795 544285 507702 505335 946100 808061 271262 54850 81069 111509 650599 710932 928248 578774 569702 646685 845482 392838 327364 571871 232194 995265 275552 286878 532960 150365 228523 983179 556523 138854 678393 300462 485582 405028 951737 33683 192145 814920 531983 399012 164582 456474 368193 665078 379999 826249 494534 180458 358907 439799 632829 309697 655451 172410 827432 656966 183334 937146 727136 335100 739417 83854 474896 770461 695984 423610 58470 712341 390300 727217 462250 852911 153311 877625 749817 873770 418458 994296 431953 348335 247436 79633 862867 260934 776112 113442 932772 522614 939456 620604 800607 137390 973752 122207 553800 690711 99772 375443 347204 262176 364286 546004 69055 39124 492186 422026 914014 954050 972479 46949 293135 308737 270845 643987 972091 189368 859374 98766 734947 288600 414581 186985 347183 955140 373510 668961 363760 117970 822518 918073 722199 684315 975876 723141 151127 867281 673708 231827 398231 479906 123034 133871 552603 806869 739093 485911 15170 751982 653792 620600 158674 277258 770254 381620 733382 887847 788846 872221 907817 188744 75695 744385 485160 751211 696612 174678 954129 227168 587694 44878 685225 200258 921354 840706 124422 244846 594245 72211 865695 625787 427059 752606 103184 457320 792706 666639 481515 445181 593306 941125 126806 766150 898995 135357 450731 343165 379401 573389 218016 758350 597044 379463 583846 377233 607389 111068 390220 490873 797189 374623 979914 672787 500820 208370 11448 403978 771713 178678 603095 550363 156248 753906 484185 666866 664062 348788 830574 318071 608619 668355 138769 907922 823510 90119 441999 211270 107979 48931 326256 320467 829105 906637 991080 60440 378104 979052 407179 137905 460724 968063 