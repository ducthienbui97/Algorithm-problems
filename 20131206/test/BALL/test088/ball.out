12354021962787
